robottelo/ui/my_account.py

```diff
@@ -10,3 +11,48 @@ class MyAccount(Base):
     def navigate_to_entity(self):
         """Navigate to My Account page"""
         Navigator(self.browser).go_to_my_account()
+
+    def _update_helper(self, input_locator_name, new_value,
```

Comment:
Please, remove all helpers in favor our usual approach which is more generic and user friendly
```
    def update(self, first_name=None, last_name=None, new_password=None, password_confirmation=None,
                locale=None, email=None......):
        """Update my account related fields from UI"""
        self.navigate_to_entity()
        if email:
            self.assign_value(locators['users.email'], email)
        if first_name:
            self.assign_value(locators['users.firstname'], first_name)
        if last_name:
            self.assign_value(locators['users.lastname'], last_name)
        if locale:
            self.select(locators['users.language_dropdown'], locale)
        if authorized_by:
            self.select(locators['users.authorized_by'], authorized_by)
        if new_password:
            self.assign_value(locators['users.password'], new_password)
        
      ### Yes, that is needed as you maybe plan to test negative scenario, 
      ### so just passing password into both field is invalid

        if password_confirmation:
            self.assign_value(
                locators['users.password_confirmation'], password_confirmation)
        self.click(common_locators['submit'])
        self.wait_until_element_is_not_visible(common_locators["notif.success"])
        self.navigate_to_entity()
        return self.wait_until_element_exists(value_locator)
```